CHANGELOG.md

```diff
@@ -216,15 +216,12 @@ All notable changes to this project will be documented in this file. The format
 ### Added
 
 - pdksync - \(MODULES-6805\) metadata.json shows support for puppet 6 [\#1026](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1026) ([tphoney](https://github.com/tphoney))
+- pdksync - \(MODULES-7705\) - Bumping stdlib dependency from \< 5.0.0 to \< 6.0.0 [\#1018](https://github.com/puppetlabs/puppetlabs-postgresql/pull/1018) ([pmcmaw](https://github.com/pmcmaw))
```

Comment:
What is this change? Why are you changing the release notes from a very old version?

Reply:
Hi @ekohl this has been generated as part of auto release work . The module needs pdk update to include the latest changes from the tool. Will be closing this PR. Thank you.